README.md

```diff
@@ -304,6 +304,33 @@ channel.presence.history { presencePage, error in
 }];
 ```
 
+### Using the `authCallback`
+
+A callback to call to obtain a signed token request.
+Assume given auth `json` data, then `ARTClientOptions` and `ARTRealtime` objects can be instantiated as follow:
+
+**Swift**
+
+```swift
+let clientOptions = ARTClientOptions()
+clientOptions.authCallback = { params, callback in
+    let tokenParams = ARTTokenParams(clientId: json["clientId"] /*string*/)
+
+    let tokenRequest = ARTTokenRequest(tokenParams: tokenParams),
+                                           keyName: json["keyName"], /*string*/
```

Comment:
The `/*string*/` comments are too noisy in my opinion. Swift is already strongly typed.